index.js

```diff
@@ -130,6 +132,11 @@ function parseRubyEngineAndVersion(rubyVersion) {
     const rubyLine = toolVersions.split(/\r?\n/).filter(e => /^ruby\s/.test(e))[0]
     rubyVersion = rubyLine.match(/^ruby\s+(.+)$/)[1]
     console.log(`Using ${rubyVersion} as input from file .tool-versions`)
+  } else if (rubyVersion === 'mise.toml') { // Read from mise.toml
+    const toolVersions = fs.readFileSync('mise.toml', 'utf8').trim()
+    const rubyLine = toolVersions.split(/\r?\n/).filter(e => /^ruby\s=\s/.test(e))[0]
+    rubyVersion = rubyLine.match(/^ruby\s=\s"(.+)"$/)[1]
```

Comment:
Looking at https://mise.jdx.dev/configuration.html it's also frequent to single-quote the version, could you allow both `'` and `"` in this regexp?